.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform
+terraform.tfstate
+terraform.tfstate.backup
+.terraform.lock.hcl
```

Comment:
Please add a newline to the end of each file. So there is no ⛔  sign displayed on GitHub.
Having newline at the end of a file is important since Linux may incorrectly process files without newlines